### DISCUSSION TITLE
Let's look ahead to v0.0.6!

### DISCUSSION THREAD

**Discussion by weykon** [weykon/new-man#8] [Ideas]
so now, what about v0.0.6 ? 
- [ ] change window size
- [ ] other ...

below it was discussion for v0.0.4 ⬇️
---

- [x] 1. click the website from the page. [issue](https://github.com/weykon/new-man/issues/5)
- [x] 2. proxy setting (at v0.0.4 double finger click will display the setting)

Let's talk about what features we want to add!

**Comment by mkultracreative** [comment]
Ability to connect to alter its own code to include diff functionality. 1) Would need to be able to loop through files on GitHub or locally to  2) make code suggestions based on relevant data

**Comment by mkultracreative** [comment]
Really just want the support for #1

**Reply by weykon** [reply]
#1  was finished.

**Reply by weykon** [reply]
oh, you mean this <ins>click the website from the page.</ins>

**Reply by weykon** [reply]
OK, I will finish that.

**Reply by weykon** [reply]
Hi!  I finished the feature you hope.
> click the website from the page.

at [v0.0.5](https://github.com/weykon/new-man/releases/tag/v0.0.5)
